templates/frontend/pages/editorialHistory.tpl

```diff
@@ -30,7 +30,7 @@
 						</span>
 						<span class="name">
 							{$mastheadUser['user']->getFullName()|escape}
-							{if $mastheadUser['user']->getData('orcid') && $mastheadUser['user']->getData('orcidAccessToken')}
+							{if $mastheadUser['user']->getData('orcid') && $mastheadUser['user']->getData('orcidAccessToken') && $mastheadUser['user']->hasVerifiedOrcid()}
```

Comment:
I think we should remove the check for the `orcidAccessToken` as the addition of `hasVerifiedOrcid` covers this better.